src/vorta/borg/version.py

```diff
@@ -22,6 +22,7 @@ def prepare(cls):
             return ret
 
         ret['cmd'] = ['borg', '--version']
+        ret['profile_name'] = 'default'
```

Comment:
Did you try setting it to `None`? `'default'` might be the name of an existing profile but such a profile doesn't have to exist possibly resulting in unexpected sideeffects.

Reply:
Setting it to `None` also seems to work, yes. I think I chose `'default'` because the constructor of `BorgJob` mentions it, but I change it if `None` is better.

Reply:
I see. Then let's stick to 'default'. However this seems like an unreasonable design choice to me.